feature-libs/user/profile/components/update-password/update-password-component.service.ts

```diff
@@ -42,10 +42,7 @@ export class UpdatePasswordComponentService {
   form: UntypedFormGroup = new UntypedFormGroup(
     {
       oldPassword: new UntypedFormControl('', Validators.required),
-      newPassword: new UntypedFormControl('', [
-        Validators.required,
-        CustomFormValidators.passwordValidator,
```

Comment:
[MEDIUM]
Was it a deliberate change? Why don't we check the validity of the new email anymore?

Reply:
Only check password validity. It's known issue as the backend also check password validity. We'll not check them in front end.